easybuild/easyconfigs/r/rpy2/rpy2-2.8.2-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'rpy2'
+version = '2.8.2'
+
+homepage = 'http://rpy.sourceforge.net/'
+description = """rpy2 is a redesign and rewrite of rpy. It is providing a low-level 
+interface to R from Python, a proposed high-level interface, including wrappers to 
+graphical libraries, as well as R-like structures and functions. """
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2c1a313df4e64236dcfe1078ce847b8e3c180656c894928d3a4b391aacb9b24c']
+
+python = 'Python'
+pyver = '2.7.13'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('R', '3.4.0', '-X11-20170314'),
+]
+
+# R-3.4.3-foss-2017b-X11-20171023.eb
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%s/site-packages/%%(name)s-%%(version)s-py%s-linux-x86_64.egg' % (pyshortver, pyshortver)],
```

Comment:
We usually check for a non-empty `site-packages` directory rather than the `.egg` directory itself, since the latter can also be a (zip) file rather than a directory.
Also, you can use the `%(pyshortver)s` template:

```python
'dirs': ['lib/python%(pyshortver)s/site-packages']
```